pkg/services/provisioning/dashboards/file_reader_test.go

```diff
@@ -32,403 +33,432 @@ const (
 var fakeService *fakeDashboardProvisioningService
 
 func TestCreatingNewDashboardFileReader(t *testing.T) {
-	Convey("creating new dashboard file reader", t, func() {
-		cfg := &config{
+	setup := func() *config {
+		return &config{
 			Name:    "Default",
 			Type:    "file",
 			OrgID:   1,
 			Folder:  "",
 			Options: map[string]interface{}{},
 		}
+	}
 
-		Convey("using path parameter", func() {
-			cfg.Options["path"] = defaultDashboards
-			reader, err := NewDashboardFileReader(cfg, log.New("test-logger"), nil)
-			So(err, ShouldBeNil)
-			So(reader.Path, ShouldNotEqual, "")
-		})
+	t.Run("using path parameter", func(t *testing.T) {
```

Comment:
I tend to leave it without a wrapper, since the parent test is named `TestCreatingNewDashboardFileReader` and it would be printed as `TestCreatingNewDashboardFileReader/Using_path_parameter` in the test output already.